### DISCUSSION TITLE
**NEW** Jellyseerr LXC

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#1657] [Announcements]
## Jellyseerr LXC

```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/jellyseerr.sh)"
```

Creates an optimized production build

**Comment by Deekerman** [comment]
How do we change it from Jellyfin to Emby?

**Reply by tteck** [reply]
This feature will be included in a future update. Please execute the commands in the Jellyseerr console.

Create a `jellyseerr.conf` file
```
systemctl stop jellyseerr
mkdir -p /etc/jellyseerr/
cat <<EOF >/etc/jellyseerr/jellyseerr.conf
PORT=5055
# HOST=0.0.0.0
JELLYFIN_TYPE=emby
EOF
```

Update the `jellyseerr.service` file
```
cat <<EOF >/etc/systemd/system/jellyseerr.service
[Unit]
Description=jellyseerr Service
After=network.target

[Service]
EnvironmentFile=/etc/jellyseerr/jellyseerr.conf
Environment=NODE_ENV=production
Type=exec
WorkingDirectory=/opt/jellyseerr
ExecStart=/usr/bin/yarn start

[Install]
WantedBy=multi-user.target
EOF
systemctl daemon-reload
systemctl start jellyseerr
```

**Comment by Deekerman** [comment]
When I first did it I got an internal server error. then I `#HOST=0.0.0.0` and reloaded. boom worked.

**Comment by Deekerman** [comment]
Thank you so much sir!

**Comment by gg-coder-dev** [comment]
After install (without any errors) when i am trying to reach http://192.168.1.217:5055 there is "Internal Server Error" in webbrowser. I tried 3 times to use script and still the same error message

**Reply by tteck** [reply]
Make modifications to the file located at `/etc/jellyseerr/jellyseerr.conf` by commenting out the line with `HOST=0.0.0.0`, like this: `# HOST=0.0.0.0`. After making this change, reboot the LXC.

**Reply by gg-coder-dev** [reply]
Thank you very much, this solution is working :)
